hack/init_control_plane_gke.sh

```diff
@@ -16,50 +16,27 @@
 
 # Usage: ./init_control_plane_gke.sh
 # The current project set in gcloud MUST be the same as where the cluster is running.
-
-NAMESPACE=cloud-run-events
-GSA_NAME=cloud-run-events
-PROJECT_ID=$(gcloud config get-value project)
-CLUSTER_NAME="$(cut -d'_' -f4 <<<"$(kubectl config current-context)")"
-KSA_NAME=controller
-
-# Enable APIs.
-gcloud services enable pubsub.googleapis.com
-gcloud services enable storage-component.googleapis.com
-gcloud services enable storage-api.googleapis.com
-gcloud services enable cloudscheduler.googleapis.com
-gcloud services enable logging.googleapis.com
-gcloud services enable cloudbuild.googleapis.com
-gcloud services enable stackdriver.googleapis.com
-gcloud services enable iamcredentials.googleapis.com
-
-# Enable workload identity.
-gcloud container clusters update ${CLUSTER_NAME} \
-  --workload-pool=${PROJECT_ID}.svc.id.goog
-
-# Modify all node pools to enable GKE_METADATA.
-pools=$(gcloud container node-pools list --cluster=${CLUSTER_NAME} --format="value(name)")
-while read -r pool_name
-do
-  gcloud container node-pools update "${pool_name}" \
-    --cluster=${CLUSTER_NAME} \
-    --workload-metadata=GKE_METADATA
-done <<<"${pools}"
-
-# Create the service account for the control plane.
-gcloud iam service-accounts create ${GSA_NAME}
-
-# Grant permissions to the service account for the control plane to manage native GCP resources.
-gcloud projects add-iam-policy-binding ${PROJECT_ID} --member=serviceAccount:${GSA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com --role roles/pubsub.admin
-gcloud projects add-iam-policy-binding ${PROJECT_ID} --member=serviceAccount:${GSA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com --role roles/storage.admin
-gcloud projects add-iam-policy-binding ${PROJECT_ID} --member=serviceAccount:${GSA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com --role roles/cloudscheduler.admin
-gcloud projects add-iam-policy-binding ${PROJECT_ID} --member=serviceAccount:${GSA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com --role roles/logging.configWriter
-gcloud projects add-iam-policy-binding ${PROJECT_ID} --member=serviceAccount:${GSA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com --role roles/logging.privateLogViewer
-gcloud projects add-iam-policy-binding ${PROJECT_ID} --member=serviceAccount:${GSA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com --role roles/iam.serviceAccountAdmin
+source $(dirname $0)/lib.sh
+readonly CLUSTER_LOCATION_TYPE="zonal"
```

Comment:
Is `CLUSTER_LOCATION_TYPE` really readonly? Line 25 sets it again.

Reply:
This is a bug. I will fix it. This variable should not be readonly